bitnami/thanos/templates/ruler/statefulset.yaml

```diff
@@ -135,6 +135,7 @@ spec:
             {{- if and $query.enabled .Values.ruler.dnsDiscovery.enabled }}
             - --query=dnssrv+_http._tcp.{{ include "common.names.fullname" . }}-query.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
             {{- end }}
+            - --alert.query-url={{ .Values.ruler.query_url }}
```

Comment:
As thanos-query service is part of the chart, I'd calculate the url asw we do for example in: https://github.com/bitnami/charts/blob/0594dcb3c0161515aa8ecf75cfba0a1f5ab468c0/bitnami/thanos/templates/NOTES.txt#L30

Maybe is worth to create a helper to avoid that ugly piece of code twice

Reply:
Hi @fmulero!
Thank you for reviewing and kind comments 🤗 

**Re: calculating the URL**
Suppose someone has a public routable domain(for example) for their Thanos query service, Now, if we calculate and provide them our k8s svc FQDN of Thanos Query, it won't work unless they have set up some kind of VPN or something to access k8s service in browser.
So the point is, if we calculate and use the value for Thanos query for  `--alert.query-url` param, we might be restricting the users for values of query-url which they can use with this setup. (because now no one will be able to set any other value than the calculated one)

thoughts?

Reply:
@fmulero Can you please check this ?

Reply:
We can keep both approaches, if the user provides the `queryURL` that value should be used. In other case, if `queryURL` is empty we should calculate it.

Reply:
Totally agree. Will do the same!